android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java

```diff
@@ -20,7 +20,7 @@
     private final ReactApplicationContext context;
 
     private int smallIcon;
-    private NotificationCompat.Action play, pause, stop, next, previous;
+    private NotificationCompat.Action play, pause, stop, next, previous, skipForward, skipBackward;
```

Comment:
I don't understand why you add new actions if you're replacing the next/previous actions